examples/gallery/ui/pages/page.slint

```diff
@@ -19,10 +20,11 @@ export component Page inherits VerticalBox {
         // Spacer
         Rectangle {}
 
-        CheckBox {
+        Switch {
             horizontal-stretch: 0;
-            text: "Disable widgets";
-            checked <=> GallerySettings.widgets-disabled;
+            text: self.checked ? "Widgets enabled" : "Widgets disabled";
```

Comment:
Hmmm, dynamic text with a switch element seems weird to me. I don't think I've ever seen that in a UI where the switch's text changes depending on its own state. Are you sure about this?

Reply:
I've changed it.